src/pages/organizations.js

```diff
@@ -157,19 +132,99 @@ export const query = graphql`
               }
             }
           }
-          Sector {
+        }
+      }
+    }
+    subOrganizations: allAirtable(
```

Comment:
I'd love to have one graphql query for fetching, both organizations that belong to the top cat and sub cat, but i really dont understand graphql well enough.

Short of that, it would be nice to use a fragment... to share organization logic, but i dont know that well enough either.

Reply:
@jasonLaster I'm working on a gatsby transformer that would inject parent categories on organizations before query time, allowing us to do this. Until then though, we could achieve both of the things you're wondering about:

* The query engine is based on [sift](https://www.npmjs.com/package/sift), which supports the `$or` operator
* Fragments work in Gatsby as well: [Using GraphQL Fragments in Gatsby](https://www.gatsbyjs.org/docs/using-graphql-fragments/)